.stylelintrc.json

```diff
@@ -8,7 +8,7 @@
     "scss/percent-placeholder-pattern": "^(denhaag|utrecht)-[a-z0-9-]+$",
     "scss/operator-no-newline-after": null,
     "scss/at-extend-no-missing-placeholder": null,
-    "custom-property-pattern": "^(denhaag|utrecht)-[a-z0-9-]+$",
+    "custom-property-pattern": "^(denhaag|_denhaag|utrecht)-[a-z0-9-]+$",
```

Comment:
How about "optional underscore" `_?`

 `"^_?(denhaag|utrecht)-[a-z0-9-]+$",`
 
 That would make it possible to extend `utrecht-button` and specify `--_utrecht-button-background-color`.

Reply:
Nice one 👍